### DISCUSSION TITLE
Is there a reason why transitions for navigation sub-headers are disabled on small viewports?

### DISCUSSION THREAD

**Discussion by rolandjitsu** [mui/toolpad#4503] [Discussions]
I'm looking at

https://github.com/mui/toolpad/blob/1561c235d7064028bceae07f169eef2db677cd2c/packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx#L251

and at https://github.com/mui/toolpad/blob/1561c235d7064028bceae07f169eef2db677cd2c/packages/toolpad-core/src/DashboardLayout/DashboardSidebarSubNavigation.tsx#L124

and I'm wondering if there's a specific reason why transitions are disabled on small viewports?

It feels a bit odd when the sidenav transitions smoothly when opening but the sub-header just snap.

https://github.com/user-attachments/assets/658bcc9f-20b7-4d53-af65-b5b5192c0a57

**Comment by apedroferreira** [comment]
The original intention was to disable the transitions only for the temporary sidebar that shows on mobile devices, but it seems like there's a bug as it can also show on the persistent sidebar...
We'll fix it, sorry and thanks for reporting it!

**Reply by rolandjitsu** [reply]
I understand. I'm glad that this is something that is planned to be fixed! Thanks.
